collections/hub/sdk-bulk/basic/reset-positions-with-settlements.json

```diff
@@ -0,0 +1,774 @@
+{
+  "name": "multi",
+  "test_cases": [
+    {
+      "id": 1,
+      "name": "Create Settlement on the current open window",
```

Comment:
Hi @sri-miriyala , as these tests include calls to switch APIs like settlements, I think we should have these tests in different folder like `advanced` instead of `basic`

Reply:
hi @vijayg10 ...I updated the structure. Can you please review.